webapp/src/utils/config.spec.ts

```diff
@@ -242,12 +244,41 @@ describe('config.ts', () => {
             '    github_token: github_123245',
           ].join('\n'),
         });
-        const config = await ServerConfig.load();
+        const config = await ServerConfig.read();
         expect(config.projects.length).toEqual(2);
         expect(config.projects[0].name).toEqual('foo');
-        expect(config.projects[0].subProjectPath).toEqual('/path/to/repo/subproject1');
+        expect(config.projects[0].subProjectPath).toEqual(
+          '/path/to/repo/subproject1',
+        );
         expect(config.projects[1].name).toEqual('bar');
-        expect(config.projects[1].subProjectPath).toEqual('/path/to/repo/subproject2');
+        expect(config.projects[1].subProjectPath).toEqual(
+          '/path/to/repo/subproject2',
+        );
+      });
+      it('throws for empty projects file', async () => {
+        expect.assertions(1);
+        mock({
+          './config/projects.yaml': '',
+        });
+        const actual = () => ServerConfig.read();
```

Comment:
On 4217140a464f5558e9c0a7e1754a7d369126a7a0, this has been resolved by storing the returned promise in actual instead of storing a function producing the promise.